packages/cli/src/commands/worker.ts

```diff
@@ -61,23 +61,6 @@ export class Worker extends BaseCommand {
 
 		try {
 			await this.externalHooks?.run('n8n.stop', []);
-
-			const hardStopTimeMs = Date.now() + this.gracefulShutdownTimeoutInS * 1000;
```

Comment:
Note that this time was misreporting, the graceful shutdown had already started at `BaseCommand.onTerminationSignal`